deployments/arbitrum-goerli/usdc/roots.json

```diff
@@ -1,7 +1,8 @@
 {
-    "comet": "0x1d573274E19174260c5aCE3f2251598959d24456",
+    "comet": "0x0C94d3F9D7F211630EDecAF085718Ac80821A6cA",
     "configurator": "0x1Ead344570F0f0a0cD86d95d8adDC7855C8723Fb",
     "rewards": "0x8DA65F8E3Aa22A498211fc4204C498ae9050DAE4",
     "bridgeReceiver": "0xAC9fC1a9532BC92a9f33eD4c6Ce4A7a54930F376",
-    "bulker": "0x987350Af5a17b6DdafeB95E6e329c178f44841d7"
+    "bulker": "0x987350Af5a17b6DdafeB95E6e329c178f44841d7",
+    "CCTPMessageTransmitter": "0x109bc137cb64eab7c0b1dddd1edf341467dc2d35"
```

Comment:
This doesn't need to be part of roots right?

Reply:
I think we can put the address to CCTP scripts instead of here. 

The current `CCTP-attestation.ts` implementation is using this field by calling `dest_dm.getContractOrThrow('CCTPMessageTransmitter');`.
Part of me thinking of putting here is that so the CCTP-attestation can support all other network (maybe AVAX) in the future as long as the destination has "CCTPMessageTransmitter" 😛

Reply:
I see, that makes sense.